packages/flutter_tools/test/general.shard/linux/linux_device_test.dart

```diff
@@ -27,125 +27,150 @@ final FakePlatform windows = FakePlatform(
   operatingSystem: 'windows',
 );
 
+final List<HostPlatform> hostPlatforms = <HostPlatform>[
+  HostPlatform.linux_x64,
+  HostPlatform.linux_arm64,
+];
+
 void main() {
 
   testWithoutContext('LinuxDevice defaults', () async {
-    final LinuxDevice device = LinuxDevice(
-      processManager: FakeProcessManager.any(),
-      logger: BufferLogger.test(),
-      fileSystem: MemoryFileSystem.test(),
-      operatingSystemUtils: FakeOperatingSystemUtils(),
-    );
-
-    final PrebuiltLinuxApp linuxApp = PrebuiltLinuxApp(executable: 'foo');
-    expect(await device.targetPlatform, TargetPlatform.linux_x64);
-    expect(device.name, 'Linux');
-    expect(await device.installApp(linuxApp), true);
-    expect(await device.uninstallApp(linuxApp), true);
-    expect(await device.isLatestBuildInstalled(linuxApp), true);
-    expect(await device.isAppInstalled(linuxApp), true);
-    expect(await device.stopApp(linuxApp), true);
-    expect(device.category, Category.desktop);
-
-    expect(device.supportsRuntimeMode(BuildMode.debug), true);
-    expect(device.supportsRuntimeMode(BuildMode.profile), true);
-    expect(device.supportsRuntimeMode(BuildMode.release), true);
-    expect(device.supportsRuntimeMode(BuildMode.jitRelease), false);
+    for (final HostPlatform hostPlatform in hostPlatforms) {
+      final LinuxDevice device = LinuxDevice(
+        processManager: FakeProcessManager.any(),
+        logger: BufferLogger.test(),
+        fileSystem: MemoryFileSystem.test(),
+        operatingSystemUtils: FakeOperatingSystemUtils(hostPlatform: hostPlatform),
+      );
+
+      final PrebuiltLinuxApp linuxApp = PrebuiltLinuxApp(executable: 'foo');
+      expect(
```

Comment:
this is an example of an expect that isn't very clear. It would be better if there were 2 test cases and each expect matched against the enum without any logic.

Reply:
I fixed unit tests.